pax/plugins/io/WaveformSimulator.py

```diff
@@ -12,6 +12,7 @@
 
 from pax import plugin, units, utils
 
+import root_pandas
```

Comment:
Actually, this is needed to endow DataFrame with the to_root function:
https://github.com/XENON1T/pax/pull/453/commits/5561d5fffe7f5d169cb2324768d538b3a9a97e42#diff-9b725188cd6edbaadd4665a5c76f69ffR57